xdsl/dialects/x86/__init__.py

```diff
@@ -0,0 +1,11 @@
+from xdsl.ir import Dialect
+
+from .register import *
```

Comment:
```suggestion
from .register import * # noqa: F405
```

Reply:
@superlopuh This is exactly what I tried in the previous commit, "removed ops.py", but this results in a failed formatting test.

Reply:
The error technically fires on line 9, and I can only assume it would fire on each individual addition to the dialect as well

Reply:
at this stage if the star makes things tedious let's import each class?

Reply:
Then I’d need to also import the register constants or otherwise the tests won’t work, but in that case it’s going to complain that the imports are unused in the init file

Reply:
can do both?

Reply:
Or have clients import them from the `x86.registers` module, that's sort of ok, since the initial proposal was `x86.Registers` anyway

Reply:
I’m fine with whatever but I thought the whole point of doing it this way was to avoid importing directly from the submodules of x86?

Reply:
I'm not 100% sure how important it is. I think it would be good to have all the operations and attributes available in `xdsl.dialects.x86`, but the register constants I think it's not critical

Reply:
alright, will do it that way then